ethcore/src/engines/authority_round/mod.rs

```diff
@@ -1516,6 +1537,15 @@ impl Engine<EthereumMachine> for AuthorityRound {
 			self.validators.report_malicious(header.author(), set_number, header.number(), Default::default());
 			Err(EngineError::DoubleVote(*header.author()))?;
 		}
+		// Report malice if the validator produced other sibling blocks in the same step.
+		let received_block_key = (header.number(), header.author().clone());
```

Comment:
I think `Address` is `Copy`:
```suggestion
		let received_block_key = (header.number(), *header.author());
```

Reply:
Thanks! I rely on `clippy` too much in such cases, and it's not usable here at all.